plugin/session_buffer.py

```diff
@@ -352,9 +353,8 @@ def _do_color_boxes_async(self, view: sublime.View, version: int) -> None:
                 self._if_view_unchanged(self._on_color_boxes_async, version)
             )
 
-    def _on_color_boxes_async(self, view: sublime.View, response: Any) -> None:
-        color_infos = response if response else []
-        self.color_phantoms.update([lsp_color_to_phantom(view, color_info) for color_info in color_infos])
+    def _on_color_boxes_async(self, view: sublime.View, response: List[ColorInformation]) -> None:
+        self.color_phantoms.update([lsp_color_to_phantom(view, color_information) for color_information in response])
```

Comment:
Ok will revert the None check, although that is agains the LSP spec and should be addressed on the server.
but it will not hurt to have a None check.